packages/dbos-cloud/organizations/organization.ts

```diff
@@ -70,3 +70,38 @@ export async function orgInvite(host: string, json: boolean) {
       return 1;
     }
   }
+
+  export async function rename(host: string, oldname: string, newname:string) {
+    const logger = getLogger();
+    const userCredentials = await getCloudCredentials();
+    const bearerToken = "Bearer " + userCredentials.token;
+    try {
+        const res = await axios.post(`https://${host}/v1alpha1/${userCredentials.organization}/organizations`, 
+          {
+            oldName: oldname,
+            newName: newname
+          },  
+          { headers: {
+              "Content-Type": "application/json",
+              Authorization: bearerToken,
+          }         
+        });
+
+        if (res.status === 204) {
+          logger.info(`Successfully renamed organization ${oldname} to ${newname}. Please logout and login to refresh before any further commands.`);
+        } else {
+          logger.error(`Failed to rename organization ${oldname} to ${newname}. Got status code ${res.status}`);
+        }
```

Comment:
I think `axios` will throw an exception if the status is not in the 200s, so you might not need to do that check

Reply:
Test with 204. Seems to be working

Reply:
yeah 204 won't throw, what I'm saying is that this branch seems unreachable

Reply:
What I'm saying is that this branch can only enter with 200s status other than 204. Do you think it could be confusing for users to read a message saying `Failure zyx: Got status code 200`? Probably this will never happen?

Reply:
So you are saying axios throws exception. if else not needed. Just log success message